scripts/nn/examples/mnist_lenet_distrib_sgd.dml

```diff
@@ -120,8 +120,8 @@ train = function(matrix[double] X, matrix[double] Y,
       # the other groups.
       group_beg = ((g-1) * group_batch_size) %% N + 1
       group_end = min(N, group_beg + group_batch_size - 1)
-      X_group_batch = X[group_beg:group_end,]
-      y_group_batch = Y[group_beg:group_end,]
+      row_group = group_beg - group_beg + 1
```

Comment:
Also, just for quick readability, I think it might be helpful to rename `row_group` to maybe `nrow_group`.